app/models/ability.rb

```diff
@@ -50,8 +50,10 @@ def not_signed_in
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can :show, User
     unless CONFIG['authentication']['ichain']['enabled']
-      can [:show, :create], Registration do |registration|
-        registration.new_record?
+      can :show, Registration
+
```

Comment:
You need to maintain the previous condition of `registration.new_record?` in the new abilities you define.